Integrations/python/test/testParquetTools.py

```diff
@@ -86,6 +87,25 @@ def testCreation(self):
                 self.assertFalse(os.path.exists(fileLocation2))
         shutil.rmtree(baseDir)
 
+    def testDecimal(self):
+        jbigdecimal = jpy.get_type('java.math.BigDecimal')
```

Comment:
Because dh.bigdecimal is not a jpy type.  Use the source, Luke.  We /could/ get to the jpy type from the dh.bigdecimal through some private method calling (like the internals of Types.py do) but... those methods are private?

Reply:
I totally read the source, I just misunderstood what it was doing in `Types.py`. Sorry for my confusion.